build-system/tasks/check-types.js

```diff
@@ -233,7 +252,7 @@ async function checkTypes() {
   cleanupBuildDir();
   maybeInitializeExtensions();
   typecheckNewServer();
-  await compileCss();
+  await Promise.all([compileCss(), compileJison()]);
```

Comment:
[This](https://docs.google.com/document/d/1vac_qbUGQ_eQiEIwhVQB5G4ASYtI509ZRhIZklJj7zw/edit) proposal by @alanorozco may make this unnecessary in future. 🤞

Reply:
🤞

Reply:
🤞